Description: increase the maximum number of kerberos realms from 32 to 512

Author: Tuomas Räsänen <tuomasjjrasanen@tjjr.fi>

--- krb5-1.10+dfsg~beta1.orig/src/kdc/main.c
+++ krb5-1.10+dfsg~beta1/src/kdc/main.c
@@ -91,7 +91,7 @@ static int rkey_init_done = 0;
 static volatile int signal_received = 0;
 static volatile int sighup_received = 0;
 
-#define KRB5_KDC_MAX_REALMS     32
+#define KRB5_KDC_MAX_REALMS     512
 
 static const char *kdc_progname;

